common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -43,7 +43,7 @@ def is_browser_on_page(self):
         """
         Check if browser is showing correct page.
         """
-        return self.q(css='body.view-profile').present
+        return self.q(css='body.view-profile .account-settings-container').present
```

Comment:
Yeah it turned out to be a flaky condition that was exposed in multiprocess mode. This is inside the body so it renders later.